gocron.go

```diff
@@ -128,27 +132,28 @@ func (j *Job) Do(jobFun interface{}, params ...interface{}) {
 func (j *Job) At(t string) *Job {
 	hour := int((t[0]-'0')*10 + (t[1] - '0'))
 	min := int((t[3]-'0')*10 + (t[4] - '0'))
-	if hour < 0 || hour > 23 || min < 0 || min > 59 {
+	sec := int((t[6]-'0')*10 + (t[7] - '0'))
+	if hour < 0 || hour > 23 || min < 0 || min > 59 || sec < 0 || sec > 59 {
```

Comment:
This does not handle leap seconds.

Reply:
Can you elaborate a bit? I suppose leap second is beyond scope of gocron lib as the job gets triggered when at specified time in seconds. After the leap second, since time resets back and job is already triggered, so no issues. 
Apologies for responding after years. :)

Reply:
The library doesn't support winter/summer time changes that happen twice per year causing trigger to be missed completely once per year or triggered at wrong hour, leap seconds is surely something much less important than that.